lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1167,6 +1292,32 @@ class TextEditingChannel {
     window._replyToPlatformMessage(callback, codec.encodeSuccessEnvelope(true));
   }
 
+  /// Used for submitting the forms attached on the DOM.
+  ///
+  /// Browser will save the information entered to the form.
+  ///
+  /// Called when the form is finalized with save option `true`.
+  /// See: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/text_input.dart#L1277
```

Comment:
Might be better to use a perma link in case that file changes:
```suggestion
  /// See: https://github.com/flutter/flutter/blob/bf9f3a3dcfea3022f9cf2dfc3ab10b120b48b19d/packages/flutter/lib/src/services/text_input.dart#L1277
```

Reply:
done.